libbuteosyncfw/common/NetworkManager.h

```diff
@@ -84,7 +92,7 @@ namespace Buteo
              *
              * @param aConnected If true, the device is online
              */
-            void valueChanged(bool aConnected);
+            void statusChanged(bool aConnected, Sync::InternetConnectionType aType);
```

Comment:
Somewhat unrelated to this change, but the ClientPlugin::connectivityStateChanged() slot should also receive this information if possible (as that would allow client plugin to decide whether or not to trigger Avatar downloading, depending on some policy -- eg, only download Avatars if WiFi connectivity is available, otherwise, only sync XML data).